cli.js

```diff
@@ -358,12 +358,17 @@ process.on('uncaughtException', function(err) {
       var sfxBundle = true;
 
     case 'bundle':
-      options = readOptions(args, ['--inject', '--yes', '--skip-source-maps', '--minify',  '--no-mangle', '--hires-source-maps', '--no-runtime']);
+      options = readOptions(args, ['--inject', '--yes', '--skip-source-maps', '--minify',  '--no-mangle', '--hires-source-maps', '--no-runtime', '--inline-source-maps']);
       if (options.yes)
         ui.useDefaults();
       options.sourceMaps = !options['skip-source-maps'];
       options.lowResSourceMaps = !options['hires-source-maps'];
       options.mangle = !options['no-mangle'];
+      
+      if (options['inline-source-maps']){
+        options.sourceMapContents = 'inline';
```

Comment:
Works only with `options.sourceMapContents` for me.
Using `options.sourceMaps` raise an exception:
```
err  SyntaxError: Unexpected token i
         at Object.parse (native)
         at new SourceMapConsumer (file:some-path/jspm_packages/npm/source-map@0.1.43/lib/source-map/source-map-consumer.js:14:24)
         at fromString (file:some-path/jspm_packages/npm/clean-css@3.1.9/lib/utils/input-source-map-tracker.js:19:26)
         at InputSourceMapStore.track (file:some-path/jspm_packages/npm/clean-css@3.1.9/lib/utils/input-source-map-tracker.js:107:54)
         at Object.whenDone (file:some-path/jspm_packages/npm/clean-css@3.1.9/lib/clean.js:76:46)
         at processNext (file:some-path/jspm_packages/npm/clean-css@3.1.9/lib/imports/inliner.js:83:115)
         at importFrom (file:some-path/jspm_packages/npm/clean-css@3.1.9/lib/imports/inliner.js:65:12)
         at processNext (file:some-path/jspm_packages/npm/clean-css@3.1.9/lib/imports/inliner.js:83:74)
         at importFrom (file:some-path/jspm_packages/npm/clean-css@3.1.9/lib/imports/inliner.js:65:12)
         at inlineLocalResource (file:some-path/jspm_packages/npm/clean-css@3.1.9/lib/imports/inliner.js:228:12)
```